.gitingore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
```

Comment:
oops.  You spelled the file name wrong!

Reply:
also start with this for Python, then add to it as needed: https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
wait, is it .DS_STORE ?